README.md

```diff
@@ -8,7 +8,7 @@ ReviOS is a customized version of Windows, designed to be lightweight, stable, a
 
 ## Features
 
-- Automaticlly built by GitHub actions for reduced release times
+- Automatically built by GitHub actions for reduced release times
```

Comment:
oh also this line is missing the implied safety benefits of auto building, since theres no human interference and the playbook is 100% like the code from the repo. find a nice way to lay that out.